Source/MediaInfo/Multiple/File_Mpeg_Descriptors.cpp

```diff
@@ -2933,12 +2933,13 @@ void File_Mpeg_Descriptors::Descriptor_59()
 {
     //Parsing
     Ztring Languages;
+    int8u subtitle_type;
     while (Element_Offset<Element_Size)
     {
         Element_Begin1("subtitle");
         int32u ISO_639_language_code;
         Get_C3 (ISO_639_language_code,                              "ISO_639_language_code");
-        Info_B1(subtitling_type,                                    "subtitling_type"); Param_Info1(Mpeg_Descriptors_component_type_O3(subtitling_type));
+        Get_B1(subtitle_type, "subtitling_type"); Param_Info1(Mpeg_Descriptors_component_type_O3(subtitle_type));

```

Comment:
```suggestion
        Get_B1 (subtitle_type,                                  "subtitling_type"); Param_Info1(Mpeg_Descriptors_component_type_O3(subtitle_type));
```

Just the formatting :)

Reply:
Thank you